src/project/project-actions.tsx

```diff
@@ -285,9 +285,19 @@ export class ProjectActions {
     }
 
     try {
-      const progress = (value: number | undefined) => {
+      const flashingCode = this.intl.formatMessage({ id: "flashing-code" });
+      const flashingMicroPython = this.intl.formatMessage({
+        id: "flashing-micropython",
+      });
+      const firstFlashNotice = (
+        <Text pb={4} fontSize="lg">
```

Comment:
Let's move the pb into the layout of the progress dialog (as mt) as it's necessary separation for anything we put above it.

Reply:
6e5da6bf20e944eaeba9274bbdeeac4ad361a05b